Cannot find module next/navigation (nuqs 2.0.0)

## Context

What's your version of `nuqs`?

```
"nuqs": "^2.0.0",
```

What framework are you using?

<!-- Keep whichever is relevant (✅: used, ❌ not used) -->

- ✅ Next.js (pages router)


Which version of your framework are you using?

<!-- Note: Next.js information can obtained by running `next info` -->

```
next: 15.0.0
```

## Description

Getting the following after installing nuqs@2.0.0 in a next.js 15 project:

```
Error: Cannot find module '/.../my-app/node_modules/next/navigation' imported from /.../my-app/node_modules/nuqs/dist/chunk-YYVIPTKH.js
Did you mean to import next/navigation.js?
```

Also occurs on a project created from scratch.

## Reproduction

1. Create new Next.js (15) project with Pages router
2. Install nuqs@2.0.0
3. Add Pages adapter
4. `npm run dev`

--- a/packages/nuqs/src/adapters/next/impl.pages.ts
+++ b/packages/nuqs/src/adapters/next/impl.pages.ts
@@ -1,4 +1,4 @@
-import { useSearchParams } from 'next/navigation'
+import { useSearchParams } from 'next/navigation.js'
 import type { NextRouter } from 'next/router'
 import { useCallback } from 'react'
 import { debug } from '../../debug'

